### DISCUSSION TITLE
Wazuh Kibana App and Integrations

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8416] [General] [answered]
Wazuh's actual differentiation from OSSEC (other than not having picked up my journald code for 2 years and then having done much of the same from scratch for unified so now the two are significantly diverged :confused: ) is in its middleware and front-end - the py code becoming the "engine" these days and the dashboard front-end to structure their telemetry in a rational human mechanism make them as useful as they are.
Is there any plan to include their dashboards/visualizations for things like AWS and O365 integration?
ping @dougburks since this is a development-related inquiry.

**Comment by dougburks** [comment, marked as answer]
> Is there any plan to include their dashboards/visualizations for things like AWS and O365 integration?

No.

> ping @dougburks since this is a development-related inquiry.

In the future, please keep the following in mind. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
_Please do not tag an individual in a discussion unless that individual has already volunteered to help you in that discussion._

**Comment by sempervictus** [comment]
Thanks,
